### PR TITLE
Updated localization

### PR DESCRIPTION
**Pull request by Thomas1664**
- added strings for vertical tab view
- translated some strings to German

### CODE REVIEW

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -984,6 +984,11 @@
   <data name="ConfirmDeleteDialogDeleteOneItem.Text" xml:space="preserve">
     <value>Are you sure you want to delete this item?</value>
   </data>
+  <data name="VerticalTabViewAddTab.Text" xml:space="preserve">
+    <value>Add tab</value>
+  </data>
+  <data name="VerticalTabViewHeader.Text" xml:space="preserve">
+    <value>Open tabs</value>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:991]:
```suggestion
    <value>Open tabs</value>
</data>
```

**Review by yair100 — CHANGES_REQUESTED** [review]
Please generate the strings using the MAT extension.

File: Files/Strings/en-US/Resources.resw (hunk 1 above)
Comment (Thomas1664) [reply Files/Strings/en-US/Resources.resw:985]: @yair100 I actually used the MAT extension but I needed to resolve merge conflicts

**Comment by Thomas1664** [comment]
@yair100 I don't know want you want me to do since I applied your suggestions during review.

**Comment by yair100** [comment]
> @yair100 I don't know want you want me to do since I applied your suggestions during review.

Can you solve the merge conflicts?

**Comment by Thomas1664** [comment]
@yair100 I can't solve the merge conflicts because Visual Studio doesn't show me the merge conflicts in the "Solve conflicts" view

**Comment by yair100** [comment]
> @yair100 I can't solve the merge conflicts because Visual Studio doesn't show me the merge conflicts in the "Solve conflicts" view

I recommend trying to solve them in Visual Studio Code.

**Comment by Thomas1664** [comment]
> I recommend trying to solve them in Visual Studio Code.

They don't show up in VS Code either

**Review by yair100 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
:shipit:

**Pull request merged**